core/config.py

```diff
@@ -0,0 +1,54 @@
+import os
+from pathlib import Path
+
+import yaml
+
+HOME_PATH = os.path.abspath(os.path.join(os.path.dirname(__file__), os.pardir))
+
+
+class Configuration:
+    ENV = os.environ.get('ENV', default='prod')
+    BASE_URL = 'https://www.cosmosid.com/'
```

Comment:
This shouldn't be hardcoded, but taken from config or better from class defaults

Reply:
Done.